### DISCUSSION TITLE
AwoX Network issues - Report sent to devs

### DISCUSSION THREAD

**Discussion by fsedarkalex** [Koenkk/zigbee2mqtt#21809] [Devices]
I wanted to drop this somewhere.... Regarding AwoX network issues...

I sent a detailled email about network issues to Awox / EGLO and received a response that they redirected it to the people in france who are responsible for the firmwares and that it is causing extensive testing and discussions already.
Well nobody knows if they ACTUALLY are working on it but maybe they are and we get some updates fixing these issues...

What I did (short summary):
I have built a Z2M based network with about 80 devices, a bit less than half of them mains powered.
12 mains devices are EGLO/AwoX and on latest firmware.

I am getting increased amounts of NO_NETWORK_ROUTE and NWK_TABLE_FULL errors, sometimes MAC_NO_ACK. Some devices - even mains powered - are dropping out of the network and need a reconnect. This especially happens to devices which are on a branch where the best route goes through an EGLO device and another route is much worse.

I also noticed it is not possible to pair a Zigbee device initially to an AwoX router.

When powering down EGLO devices and waiting a few minutes, all devices come online again and are controllable.

This is a mixture of my original tests and newer results.

I will update this thread when I get responses, maybe some of you also want to describe their observations so I can probably give this link out to AwoX at some point in the future.

**Comment by Koenkk** [comment]
Thanks for picking this up, AwoX devices are crap indeed, it's already documented on every [device page](https://www.zigbee2mqtt.io/devices/12226.html#warning-degrades-network-performance). I heard updating the device over Bluetooth solves it in some cases

**Comment by fsedarkalex** [comment]
Yes I have seen this and indeed the updates make it much better but don't solve it completely.Unfortunately EGLO has nice stuff at (more) reasonable prices compared to other brands.We bought 10 lamps before we knew about this issue.I thought maybe they were not aware about the issues and thought it could not be wrong to send them a mail. And at least it looks like they've picked it up.Hopefully it wasn't just a fin.Sent from NineVon: Koen Kanters ***@***.***>Gesendet: Freitag, 15. März 2024 20:45An: Koenkk/zigbee2mqttCc: fsedarkalex; AuthorBetreff: Re: [Koenkk/zigbee2mqtt] AwoX Network issues - Report sent to devs (Discussion #21809) 
Thanks for picking this up, AwoX devices are crap indeed, it's already documented on every device page

**Comment by fsedarkalex** [comment]
They have confirmed they were able to replicate the issue sin their lab and are currently investigating

**Comment by alteadev** [comment]
I confirm that all my problems are gone since I disconnected all my AwoX and EGLO bulbs. No more vanishing devices, no more routes issues. (Sonoff ZBDongle-P)

**Comment by fsedarkalex** [comment]
I have not heard back from EGLO since then. But I also seperated out all EGLO/AwoX devices to an own network. Interestingly they seem to work fine in their own network... No drops, no noticeable routing issues... (The error messages are still there but it works)
The only thing is that reports are sometimes not reaching Z2M (or likely are not event sent by the devices)

Will keep you updated if I hear anything back


Btw: There was a new firmware update a few weeks ago. This doesn't seem to change anything important.
